tests/unit_tests/c_wrappers/res/enkf/test_site_config.py

```diff
@@ -1,16 +1,16 @@
 import os
 
-from ert._c_wrappers.enkf import ConfigKeys, ResConfig, SiteConfig
+from ert._c_wrappers.enkf import ConfigKeys, ResConfig
 from ert._c_wrappers.enkf.enums import HookRuntime
 from ert._c_wrappers.enkf.res_config import site_config_location
 
 
 def test_constructors(snake_oil_case):
     ert_site_config = site_config_location()
     ert_share_path = os.path.dirname(ert_site_config)
-
-    assert snake_oil_case.resConfig().site_config == SiteConfig.from_config_dict(
+    res_config = ResConfig(
```

Comment:
Should probably merge `test_site_config.py` with `test_res_config.py`.

Reply:
I thought about it, but was unsure. Is it trying to actually test that we can read the actual site config file? Or is it trying to test the behavior of a selection of lines of code which have now moved?